codebuild/canary/CanaryWrapper_Classes.py

```diff
@@ -0,0 +1,1159 @@
+            self.application_process = None
+            self.print_stdout()
+        else:
+            self.print_message(
+                "[ApplicationMonitor] ERROR - cannot stop monitor application because no process is found!")
+
+    def print_stdout(self):
+        # Print the STDOUT file
+        if (os.path.isfile(self.stdout_file_path)):
+            self.print_message("Just finished Application STDOUT: ")
+            with open(self.stdout_file_path, "r") as stdout_file:
+                self.print_message(stdout_file.read())
+            os.remove(self.stdout_file_path)
+
+    def monitor_loop_function(self):
+        if (self.application_process != None):
+
+            application_process_return_code = None
+            try:
+                application_process_return_code = self.application_process.poll()
+            except Exception as e:
+                self.print_message(
+                    "[ApplicationMonitor] ERROR - exception occurred while trying to poll application status!")
+                self.print_message("[ApplicationMonitor] Exception: " + str(e))
+                self.error_has_occurred = True
+                self.error_reason = "Exception when polling application status"
+                self.error_code = 1
+                return
+
+            # If it is not none, then the application finished
+            if (application_process_return_code != None):
+
+                self.print_message(
+                    "[ApplicationMonitor] Monitor application has stopped! Processing result...")
+
+                if (application_process_return_code != 0):
+                    self.print_message(
+                        "[ApplicationMonitor] ERROR - Something Crashed in Canary/Application!")
+                    self.print_message(
+                        "[ApplicationMonitor] Error code: " + str(application_process_return_code))
+
+                    self.error_has_occurred = True
+                    self.error_reason = "Canary application crashed!"
+                    self.error_code = application_process_return_code
+                else:
+                    # Should we restart?
+                    if (self.wrapper_application_restart_on_finish == True):
+                        self.print_message(
+                            "[ApplicationMonitor] NOTE - Canary finished running and is restarting...")
+                        self.restart_monitoring()
+                    else:
+                        self.print_message(
+                            "[ApplicationMonitor] Monitor application has stopped and monitor is not supposed to restart... Finishing...")
+                        self.error_has_occurred = True
+                        self.error_reason = "Canary Application Finished"
+                        self.error_code = 0
+            else:
+                self.print_message(
+                    "[ApplicationMonitor] Monitor application is still running...")
```

Comment:
Let's remove this. It prints a wall of this in logs before we see the useful data snapshots in between.